Tests/SessionTests.swift

```diff
@@ -0,0 +1,148 @@
+import Embassy
+@testable import Turbo
+import WebKit
+import XCTest
+
+private let defaultTimeout: TimeInterval = 10000
+private let turboTimeout: TimeInterval = 30
+
+class SessionTests: XCTestCase {
+    private static let eventLoop = try! SelectorEventLoop(selector: try! KqueueSelector())
```

Comment:
Any downside in moving this `try!` heavy setup code into the throwing `setup()` method, so `eventLoop` also becomes a `var eventLoop: SelectorEventLoop!`?

Reply:
If this wasn't a static/class-level property then definitely! But see: https://github.com/hotwired/turbo-ios/pull/139#discussion_r1390233577

Reply:
Disregard, fixed with a non-static server!